tools/tcpstates.py

```diff
@@ -132,6 +134,9 @@
     u16 family = args->family;
     FILTER_FAMILY
 
+    // workaround to avoid llvm optimization which will cause context ptr args modified
+    u32 tcp_new_state = args->newstate;
```

Comment:
The `newstate` field should be `int`, other than that, LGTM.

Reply:
modify `tcp_newstate` and corresponding fields in bpf events from `unsigned int` to `int`